src/scss/base/_fonts.scss

```diff
@@ -1,4 +1,4 @@
-@include font-face(OpenSans, '#{$static}/fonts/opensans-regular', $weight: 400, $style: null, $exts: woff2 woff);
-@include font-face(OpenSans, '#{$static}/fonts/opensans-bold', $weight: 600, $style: null, $exts: woff2 woff);
+@include font-face(OpenSans, '#{$static}/fonts/opensans-regular', $weight: 200, $style: null, $exts: ttf);
```

Comment:
I think we probably want to keep both woff and ttf formats and only apply to monospaced numerals format to ordered lists and table cells?